ansible/roles/linux-webconsole/files/guacamole-playbook.yml

```diff
@@ -48,17 +48,26 @@
             # generating ]]>, which will still break XML. Therefore remove ">" from special
             # characters.
             special_chars: '!"#$%&()*+,-./:;<=?@[\]^_`{|}~'
-        
-        - block:
-          - name: Get Guacamole user info
-            getent:
-              database: passwd
-              key: "{{ guacamole_user }}"
+          
+        - name: Get Guacamole user info
+          getent:
+            database: passwd
+            key: "{{ guacamole_user }}"
```

Comment:
Same as above, I don't care which we do but we need to be consistent between the image and the playbook. Probably using a fixed `azimuth` username is easier for now, but don't pretend it can be changed in the playbook patch.

Reply:
Hardcoded for now in [b5aba01](https://github.com/azimuth-cloud/azimuth-images/pull/251/commits/b5aba01be7f28edc1b8cd5490c3ec2fa1c9821fa)